source/io_netcdf.F90

```diff
@@ -2177,47 +2168,17 @@ subroutine write_nstd_netcdf(data_file,field_id,            &
                   case(2)
                      supported = .false.
                   case(3)
-                     if (my_task == ioroot) then
-                       do n=1,ndims-1
-                         nout(n) = size(indata_2d_r4,DIM=n)
-                       enddo
-                       tb = io_dims(ndims)%start
-                       allocate (outdata_3d_r4(nout(1),nout(2),tb:tb))
-                       outdata_3d_r4(:,:,tb) = indata_2d_r4(:,:)
-                     else
-                       allocate (outdata_3d_r4(1,1,1))
```

Comment:
If you are getting rid of these lines, do we still need `outdata_3d_r4` in this routine? It looks like we can get rid of the `outdata_*_r4` in lines 2073 - 2083.